exchanges/okx/okx_websocket.go

```diff
@@ -16,7 +16,6 @@ import (
 
 	"github.com/buger/jsonparser"
 	gws "github.com/gorilla/websocket"
-	"github.com/thrasher-corp/gocryptotrader/common"
```

Comment:
```
[ERROR] | WEBSOCKET | 01/12/2025 10:29:32 | Okx websocket: connectionMonitor shutdown err: Okx websocket cannot shutdown: websocket is not connected
```
Not super worried with this one (but would prefer no problems). I've already asked it to shutdown

Reply:
I did a quick count and check and even if all available pairs are subscribed its around 5205 subs / 240 subs per connection its around 22 connections. 

The culprit which I think could be more probable of causing disconnection is to do with our `websocket routine manager/engine` processing which causes a lot of latency. Trying to log everything to stdout etc. 

I will add in the additional field, no reason why we can't screen that out.